src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'm' ? 'wife' : 'husband';
```

Comment:
2. [CODE STYLE] - Readabily is everything. Format ternary operator operands correctly - move each operand to the separate line